CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+<!--
+
+    Copyright (c) 2019-present Sonatype, Inc.
+    This program and the accompanying materials are made available under
+    the terms of the Eclipse Public License 2.0 which accompanies this
+    distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+
+-->
+
+# How to be a contributor to this project
+
+## Talk to a human first
+
+### For Sonatypers
+
+Looking to dive in? FANTASTIC! Reach out to one of our experts and we can help get you started in the right direction.
+
+* Hit us up on our Slack channel #react-components.
```

Comment:
#integrations instead.

Reply:
needs to be a public channel, would suggest we just omit this part

Reply:
It is in the sonatypers section. 👍

Reply:
> 
> 
> #integrations instead.

d'oh, missed that one! Thanks, fixed.